.editorconfig

```diff
@@ -3,10 +3,7 @@ root = true
 
 [*]
 end_of_line = lf
-
```

Comment:
Please don't remove `[*.{fs,fsi,fsx}]`, honestly rule of thumb change as little as possible.

Reply:
These settings make sense for all the files but ok